src/MediaCollections/Models/Concerns/PathUuidSafe.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Spatie\MediaLibrary\MediaCollections\Models\Concerns;
+
+use Illuminate\Support\Facades\Storage;
+use Spatie\MediaLibrary\MediaCollections\Models\Media;
+use Spatie\MediaLibrary\Support\PathGenerator\PathGeneratorFactory;
+
+trait PathUuidSafe
```

Comment:
Please rename this to `SupportsUuidInPaths`

Reply:
Renamed to `MovesOnUpdate`. Is it ok?